### DISCUSSION TITLE
Upload file size limit for GitHub community edition

### DISCUSSION THREAD

**Discussion by mobiveil32** [community/community#23287] [New to GitHub]
<p><img src="https://user-images.githubusercontent.com/63840104/181076042-2905105f-4b5d-40ea-80b5-c8e73740c9cb.jpeg" alt="Upload_error_in_GitHub.jpg" data-base62-sha1="uw7yMs3h9xNqv48j9d98OlVNpmW" width="575" height="253"></p>
<p>I am using a GitHub Community edition for our project. Not able to upload 300MB size of file in GitHub, getting above error message. </p>
<p>Please help on this. Thanks in advance.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hi <a class="mention" href="/mobiveil32">@mobiveil32</a> , </p>
<p>Thanks for posting in the GitHub Community forum. </p>
<p>GitHub has a <a href="https://help.github.com/en/github/managing-large-files/conditions-for-large-files#blocked-pushes-for-large-files" rel="nofollow noopener">100mb upload limit</a> for files. For larger files we reccomend using <a href="https://git-lfs.github.com/" rel="nofollow noopener">Git LFS storage</a> which is better suited to handle larger files. </p>
<p>I hope this helps clarify things. If you need any assistance with Git LFS please reach out! </p>

**Comment by mobiveil32** [comment]
<p>Thanks Robert. </p>
<p>Tried LFS to push the file and got below error message. The compressed file size is around 3GB.</p>
<p>Is there any file upload size limit on LFS ? My account is Git Hub community edition.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/3b3e664c4077c723cbb50cc8a95a3bfe640227a4.jpeg" data-download-href="/uploads/short-url/8s5RoMlOclUrr9c24Ayb38ku1Fy.jpg?dl=1" title="Annotation 2020-04-25 170513.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/63840104/181076043-dba88686-dc36-4119-9966-b8a1bbf762fe.jpg" alt="Annotation 2020-04-25 170513.jpg" data-base62-sha1="8s5RoMlOclUrr9c24Ayb38ku1Fy" width="690" height="50" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/3b3e664c4077c723cbb50cc8a95a3bfe640227a4_2_690x50.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/3/3b3e664c4077c723cbb50cc8a95a3bfe640227a4_2_1035x75.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/3b3e664c4077c723cbb50cc8a95a3bfe640227a4_2_1380x100.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/3b3e664c4077c723cbb50cc8a95a3bfe640227a4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2020-04-25 170513.jpg</span><span class="informations">1781×131 45.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
